ui/raidboss/timeline.js

```diff
@@ -119,6 +120,23 @@ class Timeline {
         continue;
       }
 
+      match = line.match(/^color\s+\"([^"]+)\"\s+(#[a-fA-F0-9]{6})$/);
```

Comment:
Hmm.  I think I would prefer if these were more like timeline triggers than new timeline line commands.

Maybe something like:
```
timelineStyles: [
  {
    regex: /something/,
    style: {
      color: 'papaya-whip',
      font-size: 'gigantic',
    },
  },
],
```

This would make it easier to have an array of styles, a more complicated regex to apply to lines, and to arbitrarily allow for any kind of styling rather than just a few whitelisted ones.

What do you think?

Reply:
Seems better yes, i've no clue on how to access it tho. And help ?

Reply:
It should work now